EventGenerator/src/RPCGun_module.cc

```diff
@@ -83,16 +83,22 @@ namespace mu2e {
     RandomUnitSphere   randomUnitSphere_;
     CLHEP::RandGeneral randSpectrum_;
 
-    const double            czmin_ = -1;
-    const double            czmax_ = 1;
-    const double            phimin_ = 0;
-    const double            phimax_ = CLHEP::twopi;
+    static constexpr double            czmin_ = -1;
+    static constexpr double            czmax_ = 1;
+    static constexpr double            phimin_ = 0;
+    static constexpr double            phimax_ = CLHEP::twopi;
```

Comment:
Please use 2.*M_PI .

Reply:
Please let me know if you prefer 2*M_PI or 2.*pi_v (from numbers)

Reply:
RandomUnitSphere.hh uses CLHEP::twopi

Reply:
the constants have been removed in the new commit. Do you want to modify RandomUnitSphere.hh ?